api/src/main/java/org/openmrs/module/aijar/api/deploy/bundle/CommonMetadataBundle.java

```diff
@@ -56,5 +56,6 @@ public void install() throws Exception {
         install(EncounterTypes.PNC_ENCOUNTER_TYPE);
         install(EncounterTypes.SMC_FOLLOW_UP_ENCOUNTER);
         install(encounterRole(EncounterRoles.ASSISTANT_CIRCUMCISER_NAME,EncounterRoles.ASSISTANT_CIRCUMCISER_DESCRIPTION,EncounterRoles.ASSISTANT_CIRCUMCISER_UUID));
+        install(EncounterTypes.VIRAL_LOAD_NON_SUPPRESSED);
```

Comment:
Put the encounter types together rather than having them broken up by the Encounter Role above